tests/integration-tests/tests/schedulers/test_slurm/test_slurm_memory_based_scheduling/pcluster.config.update_scheduling.yaml

```diff
@@ -0,0 +1,30 @@
+Image:
+  Os: {{ os }}
+HeadNode:
+  InstanceType: {{ instance }}
+  Networking:
+    SubnetId: {{ public_subnet_id }}
+  Ssh:
+    KeyName: {{ key_name }}
+  Iam:
+    AdditionalIamPolicies:
+      - Policy: arn:aws:iam::aws:policy/AmazonSSMManagedInstanceCore
+Scheduling:
+  Scheduler: slurm
+  SlurmSettings:
+    EnableMemoryBasedScheduling: true
```

Comment:
May be out of scope for this PR, but it would be better to have a unique config file with template parameters instead of having two with only one parameter changing.

Reply:
You are probably right. I followed the same structure as other integration tests (`test_slurm_config_update`) also to reuse some of the functions already available.

But yes, I believe this is out of scope.

Reply:
Agree it is out of scope

Reply:
Ok, we decided this is out of scope.